.ado/publish.yml

```diff
@@ -8,10 +8,10 @@ trigger:
   batch: true
   branches:
     include:
-      - master
-      - 0.58-vnext-stable
+    # Avoid publishing master builds until we're on 0.61 to avoid version collisions with 0.60-stable
```

Comment:
I was curious if comments were allowed here, but a YAML linter seems to think this is okay.